allennlp/modules/transformer/transformer_block.py

```diff
@@ -10,6 +10,24 @@
 
 
 class TransformerBlock(TransformerModule, FromParams):
+    """
+    This module is the basic transformer block, which acts as an encoder.
```

Comment:
It's possible I have this terminology wrong, but why do you call this an "encoder"? What does it encode?

I would understand the terminology if this took word piece ids in its input, because those get encoded, but this takes vectors and returns vectors.

Is that what it's called in Devlin et al?